src/Workspaces/CSharp/Portable/Extensions/BaseMethodDeclarationSyntaxExtensions.cs

```diff
@@ -6,17 +6,17 @@ namespace Microsoft.CodeAnalysis.CSharp.Extensions
 {
     internal static class BaseMethodDeclarationSyntaxExtensions
     {
-        public static BaseMethodDeclarationSyntax WithSemicolonToken(this BaseMethodDeclarationSyntax node, SyntaxToken token)
+        public static BaseMethodDeclarationSyntax WithSemicolonToken(this BaseMethodDeclarationSyntax node, SyntaxToken semicolonToken)
```

Comment:
I renamed this parameter to exactly match the one in MethodDeclarationSyntax.WithSemicolonToken and the other methods

Reply:
BTW, why do these extension methods ignore nulls?

Reply:
> BTW, why do these extension methods ignore nulls?

As per the gitter conversation: to make it less unpleasant to transform trees and have to up front deal with all the null possible nodes in a tree